src/applications/static-pages/view-modify-dependent/686-cta/containers/Form686CTA.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+import EbenefitsLink from 'platform/site-wide/ebenefits/containers/EbenefitsLink';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+import { CallToActionWidget } from 'platform/site-wide/cta-widget';
+
+const Form686CTA = props => {
+  let content = <></>;
+  if (props.showContent === undefined) {
+    content = <LoadingIndicator message="Loading..." />;
+  } else if (props.showContent === false) {
+    content = (
+      <EbenefitsLink
+        path="ebenefits/about/feature?=dependent-compensation"
+        className="usa-button"
+      >
+        Go to eBenefits to add or modify a dependent
+      </EbenefitsLink>
```

Comment:
Does this `EbenefitsLink` for sure still work? It doesn't need to be wrapped in a `Provider` or have the `store` passed as a prop? That was a runtime error last time.

Reply:
@ncksllvn - it's passed as a prop [here](https://github.com/department-of-veterans-affairs/vets-website/pull/13604/files/e9e98ba6678c3d48d1f409e94b208fb557e60968#diff-0bdca92bc824e65b7d6a2a45a226261fR10) Is that sufficient?

Reply:
If this EbenefitsLink renders okay, then yeah, totally! But since there isn't a parent `<Provider/>` and IIRC the `EbenefitsLink` is a container, it's going to run into an error about not being able to find the store

Reply:
Gotcha. I'm going to submit another PR that wraps everything in a `<Provider/>`, similar to what you did with the view dependents widget you created a little while ago.

Reply:
@ncksllvn - created a follow up pull request to wrap the CTA in a provider here - https://github.com/department-of-veterans-affairs/vets-website/pull/13613